### PR TITLE
M6 INT same url another stream

### PR DESCRIPTION
**Pull request by dct-infra**
M6 INT same url another stream

### CODE REVIEW

File: channels/fr.m3u [hunk 1]
```diff
@@ -149,9 +149,11 @@ http://live.lachainenormande.fr/live/lcn/livestream/playlist.m3u8
 https://lci-hls-live-ssl.tf1.fr/lci/1/hls/live_2328.m3u8
 #EXTINF:-1 tvg-id="LCP.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.postimg.cc/rwBVxjj2/lcp.png" group-title="Legislative",LCP (480p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/xji3qy
+#EXTINF:-1 tvg-id="M6.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 (1080p)
+https://shls-m6-france-prod-dub.shahid.net/out/v1/c8a9f6e000cd4ebaa4d2fc7d18c15988/index.m3u8
 #EXTINF:-1 tvg-id="M6.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 (720p) [Not 24/7]
 http://flusonic-4.platinum-tv.com/M6/mono.m3u8?token=test
-#EXTINF:-1 tvg-id="M6International.fr" tvg-country="FR;AD;BE;LU;MC;CH" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 International (1080p)
+#EXTINF:-1 tvg-id="M6International.fr" tvg-country="FR;AD;BE;LU;MC;CH" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 International (1080p) [Offline]
```
Comment (Dum4G) [inline channels/fr.m3u:156]: Says Signal Lost but it's not offline

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/fr.m3u (hunk 1 above)
Comment (dct-infra) [reply channels/fr.m3u:156]: Signal lost appeared later. I'll keep checking and see how it goes.

Comment (Dum4G) [reply channels/fr.m3u:156]: Stuck at this frame now
image

Comment (Sphinxroot) [reply channels/fr.m3u:156]: Capture
working on my side

Comment (LaneSh4d0w) [reply channels/fr.m3u:156]: I have "Signal Loss Detected sv1"

Comment (dct-infra) [reply channels/fr.m3u:156]: > Stuck at this frame now image

@Dum4G That's what I first saw. How do we tag the link?

Comment (dct-infra) [reply channels/fr.m3u:156]: > Capture working on my side

@Sphinxroot Weird, but that's M6 music not M6 International.

Comment (Dum4G) [reply channels/fr.m3u:156]: I think there's no need to have a placeholder link in the playlist (M6International.fr) at least until it gets fixed and M6.fr streams regular M6 now.

**Review by BellezaEmporium — DISMISSED** [review]
You'll need to fix conflict before merging.

**Pull request closed without merging**